infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -468,6 +468,15 @@ public long getSequencerEpoch() {
         return epoch == null ? Layout.INVALID_EPOCH : epoch;
     }
 
+    /**
+     * Get the sequencer cache size.
+     *
+     * @return Sequencer cache size
+     */
+    public long getSequencerCacheSize() {
+        return Long.parseLong(getServerConfig(String.class, "--sequencer-cache-size"));
```

Comment:
It colud be an NPE here, please add more checks

Reply:
There won't be NPE here. See https://github.com/CorfuDB/CorfuDB/blob/master/infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java#L134 
We provide a default value if user not specified.

Reply:
What if tomorrow someone deletes that default value?
Your code must work in any case, it must be independent of some other ode somewhere, don't rely heavily on only current code, think about changes